tests/test_react_auth_defs.py

```diff
@@ -52,11 +55,45 @@
 DOMAIN = "some-domain"
 # Mock out the call which create_authtoken makes Envs.get_user_auth_info
 # to get the allowed environments and first/last name for the user.
-ENVS.get_user_auth_info = lambda email: (ALLOWED_ENVS, FIRST_NAME, LAST_NAME)
+ENVS.get_user_auth_info = lambda email, raise_exception: (ALLOWED_ENVS, FIRST_NAME, LAST_NAME)
 
 AUTH = Auth(AUTH0_CLIENT_ID, AUTH0_SECRET, ENVS)
 
 
+class MockChaliceApp:
+
+    class MockReactApi:
+        def __init__(self):
+            self.react_authorize = AUTH.authorize
+            self.create_success_response = ReactApi.create_success_response
+            self.create_response = ReactApi.create_response
+            self.get_site_name = lambda: "some-site-name"
+            self.APP_PACKAGE_NAME = "foursight"
```

Comment:
Is this different than:
```
    class MockReactApi:
        react_authorize = AUTH.authorize
        create_success_response = ReactApi.create_success_response
        create_response = ReactApi.create_response
        get_site_name = lambda: "some-site-name"
        APP_PACKAGE_NAME = "foursight"
```
That is, does this state ever change? An init method needs to set up instance/self variables only if you need the values to be different on a per-instance basis. Also, there's usually very little value to making a class in a class. Unless you have other MockReactApi's you're trying to keep clear, it might be easier to put it at file top level and then name it as `MockReactApi` rather than `MockChaliceApp.MockReactApi`.